k8s/coordinator/development/config.toml

```diff
@@ -34,3 +34,9 @@ db = "metrics"
 # `XAYNET_REDIS__URL` depends on the enviroment variable `REDIS_AUTH`,
 # which is defined as a Kubernetes secret and exposed to the coordinator pod.
 # See: k8s/coordinator/base/deployment.yaml
+
+[s3]
+# The access_key and secret_access_key are configured via the enviroment variables
+# `XAYNET_S3__ACCESS_KEY` and `XAYNET_S3__SECRET_ACCESS_KEY`.
+# See: k8s/coordinator/base/deployment.yaml
+region = ["minio", "http://minio:9000"]
```

Comment:
What exactly is with passing the `region`? Is this the official way of informing the endpoint of the S3 endpoint?
I would assume that there would be at least 4 parameters:
* Bucket
* Endpoint
* accessKey
* secretKey

Reply:
Yes, that is the [official way](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#create-bucket-intro) 
The endpoint is implicit given via the [region code](https://docs.aws.amazon.com/general/latest/gr/rande.html). However if you don't use S3 directly but an AWS-compatible services then you have to specify the endpoint.

I added the the bucket name to the settings but made them optional (uses a default value if not present)